public/smartbanner.html

```diff
@@ -10,6 +10,7 @@
     <meta name="smartbanner:button" content="VIEW">
     <meta name="smartbanner:close-label" content="Close">
     <meta name="smartbanner:exclude-user-agent-regex" content="^.*(Windows NT|Intel Mac OS X).*$">
+    <meta name="smartbanner:disable-positioning" content="true" />
```

Comment:
is this what stops the flickering? i don't really understand this `disable-positioning` prop

Reply:
typically it adds the smartbanner to body then takes html and forces it to be positioned below the banner but that's weird because body is in html and it fucks up all our styling.

Reply:
disable-positioning stops that and just plops it in the body tag and lets us style it